src/goto-programs/remove_instanceof.cpp

```diff
@@ -28,16 +28,17 @@ class remove_instanceoft
     class_hierarchy(symbol_table);
   }
 
-  // Lower instanceof for a single functions
+  // Lower instanceof for a single function
   bool lower_instanceof(goto_programt &);
```

Comment:
Question to compiler experts: is "lowering" a well-defined technical term? I would have used "translate" or "rewrite".

Reply:
Yes, for example http://www.drdobbs.com/architecture-and-design/so-you-want-to-write-your-own-language/240165488?pgno=2 -- "rewriting more complex semantic constructs in terms of simpler ones", here "instanceof" in terms of classid equality comparisons.

Reply:
Not that I consider Dr Dobbs an authoritative reference, but ok. But then you made me search and it seems that GCC and LLVM do use that terminology. So maybe not Dragon-book style, but apparently common in practice.